src/metrics/kubernetes.py

```diff
@@ -31,6 +37,39 @@ def create_connection(self, address, *args, **kwargs) -> socket.socket:
 
         return pf.socket(port)
 
+    @staticmethod
+    def download_logs_from_pod_asyncable(kube_config: str, namespace: str, pod_name: str,
+                                         location: str):
+        kube_client = kubernetes.config.new_client_from_config(kube_config)
```

Comment:
Why not make the function a `method` and use `self._kube_client`?

Reply:
Discussed in the offsite. If you have any other idea, let me know. For now, I think this is the “best” I have in mind.